index.bs

```diff
@@ -537,7 +537,32 @@ The <dfn attribute for="XRSession">environmentBlendMode</dfn> attribute returns
 
 NOTE: Most Virtual Reality devices exhibit {{XREnvironmentBlendMode/opaque}} blending behavior. Augmented Reality devices that use transparent optical elements frequently exhibit {{XREnvironmentBlendMode/additive}} blending behavior, and Augmented Reality devices that use passthrough cameras frequently exhibit {{XREnvironmentBlendMode/alpha-blend}} blending behavior.
 
-Each {{XRSession}} has a <dfn for="XRSession">viewer reference space</dfn>, which is an {{XRReferenceSpace}} of type {{XRReferenceSpaceType/viewer}} with an [=identity transform=] [=XRSpace/origin offset=]. The [=XRSession/viewer reference space=] has a <dfn for="XRSession/viewer reference space">list of views</dfn>, which is a [=/list=] of [=view=]s corresponding to the views provided by the [=/XR device=].
+Each {{XRSession}} has a <dfn for="XRSession">output canvas</dfn>, which is an {{HTMLCanvasElement}}. The [=XRSession/output canvas=] is the DOM element that will display any content rendered for an {{XRSessionMode/inline}} {{XRSession}}. 
```

Comment:
I'm not sure if I grok what you're referring to re: "a strong statement than I'd been expecting" here. If it's the requirement that an output canvas be present in order for rAFs to process, today we already require that the inline context has an `outputContext`.

Also, To clarify: When `XRPresentationContext` (or similar) is re-introduced in the future it would function as one possible source of an output canvas. So the message here is ultimately that the output canvas is the thing that output will eventually show up on, whether via this "direct output" mechanism or by an `XRPresentationContext`-style interface.

Reply:
Ah, sorry I wasn't clear.  I was surprised that we're stating that a session always has an output canvas, even if it's null for immersive sessions.  I wasn't clear that this was strictly necessary or if instead we could just refer to the properties it was computed off of.  Not a big deal, tbh.

Reply:
It's primarily a convenience reference for the rest of the spec text to point back to and say "I'm computing (width/height/position/etc) off of this thing, which will eventually have a couple of different ways to set it.